File: my-solutions/0777-toeplitz-matrix/solution.scala
object Solution {
    def isToeplitzMatrix(matrix: Array[Array[Int]]): Boolean = {
        for(r <- 0 until matrix.length; c <- 0 until matrix(0).length){
            if(r > 0 && c > 0 && matrix(r)(c) != matrix(r-1)(c-1))
                return false
        }
        return true
    }
}

File: my-solutions/0777-toeplitz-matrix/solution.py
class Solution:
    def isToeplitzMatrix(self, matrix: List[List[int]]) -> bool:

        for i in range(len(matrix) - 1): ## Rows

            for j in range(len(matrix[0]) - 1): ## Columns

                if matrix[i][j] != matrix[i+1][j+1]:
                    return False

        return True
        
